perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -533,6 +533,13 @@ sub inspect : Private {
 
         $c->cobrand->call_hook(report_inspect_update_extra => $problem);
 
+        $c->forward('/photo/process_photo');
+        if ( my $photo_error  = delete $c->stash->{photo_error} ) {
+            $valid = 0;
+            $c->stash->{errors} ||= [];
```

Comment:
I don't think this line is necessary (it will autovivify into an array if undefined), but I see a few other clauses above use it too, and some don't.